src/ORM/DataObject.php

```diff
@@ -3090,8 +3090,11 @@ public static function get(
      *
      * @return DataObject|null The first item matching the query
      */
-    public static function get_one($callerClass, $filter = "", $cache = true, $orderby = "")
+    public static function get_one($callerClass = null, $filter = "", $cache = true, $orderby = "")
```

Comment:
I wonder if this signature change would be breaking

Reply:
It is 😞

Reply:
It _is_ backwards compatible though - is that enough?

Reply:
True - sigh. So PR against 5 then?